### PR TITLE
alex/emgg

### PR DESCRIPTION
**Pull request by alexweininger**
- emgg page
- styling
- WIP
- Make it easy to switch banner types
- Touch ups to UI

### CODE REVIEW

File: apps/next/pages/banner.tsx [hunk 1]
```diff
@@ -173,6 +201,7 @@ export default function Page({ banner }: Props) {
     const [reAuth, setReAuth] = useState(false);
 
     const BackgroundTemplate = BackgroundTemplates[bgId];
+    console.log(ForegroundTemplates);
```
Comment (AndrewJLang) [inline apps/next/pages/banner.tsx:204]: remove this

File: libs/remotion/templates/src/lib/foregrounds/Emgg.template.tsx [hunk 2]
```diff
@@ -2,12 +2,16 @@ import { ForegroundComponents } from '@pulsebanner/remotion/components';
 import { ForegroundForms } from '@pulsebanner/remotion/forms';
 import { Layer } from '../Layer';
 
-export const TwitchStream: Layer<typeof ForegroundComponents.TwitchStream> = {
```
Comment (AndrewJLang) [inline libs/remotion/templates/src/lib/foregrounds/Emgg.template.tsx:5]: what were we using this file for before?

File: apps/next/pages/emgg.tsx [hunk 3]
```diff
@@ -0,0 +1,444 @@
...
+
+    const styles: BoxProps = {
+        background: 'whiteAlpha.100',
+    };
+
+    const [isToggling, { on, off }] = useBoolean(false);
+
+    useEffect(() => {
+        axios.post('/').then(() => {
+            console.log('attempt to clear 301 redirect from cache');
```
Comment (AndrewJLang) [inline apps/next/pages/emgg.tsx:193]: remove this or change to datadog

Comment (AndrewJLang) [reply apps/next/pages/emgg.tsx:193]: Actually we don't need this thing at all

File: libs/remotion/templates/src/lib/foregrounds/Emgg.template.tsx (hunk 2 above)
Comment (alexweininger) [reply libs/remotion/templates/src/lib/foregrounds/Emgg.template.tsx:5]: nothing

File: apps/next/pages/emgg.tsx (hunk 3 above)
Comment (alexweininger) [reply apps/next/pages/emgg.tsx:193]: Yeah, probably can remove it now

**Pull request merged**